x-pack/heartbeat/scenarios/scenarios.go

```diff
@@ -70,30 +69,5 @@ var Scenarios = &ScenarioDB{
 				}, func() {}, nil
 			},
 		},
-		{
-			Name: "simple-browser",
```

Comment:
Temporarily remove browser test from scenarios. This is a very new test that will come 
back in an improved form in https://github.com/elastic/beats/pull/30632